lms/views/api/exceptions.py

```diff
@@ -155,14 +161,13 @@ def external_request_error(self):
         )
 
     @exception_view_config(context=OAuth2TokenError)
-    def oauth2_token_error(self):
-        return self.error_response()
+    def oauth2_token_error(self):  # pylint:disable=no-self-use
+        return ErrorResponseBody()
```

Comment:
Should the error have some kind of default for error code at least? It seems strange to return an entirely empty dict?

Reply:
See other comment: <https://github.com/hypothesis/lms/pull/3693#issuecomment-1040654000>. The API design is quite brilliant, I'm sure you'll agree